src/components/ContentSidebar/Sidebar.js

```diff
@@ -60,6 +64,8 @@ const Sidebar = ({
             appElement={appElement}
             rootElement={rootElement}
             onInteraction={onInteraction}
+            onDescriptionChange={onDescriptionChange}
+            descriptionTextareaProps={descriptionTextareaProps}
         />
     );
 
```

Comment:
While you are at it, remove the text under activity tab below.
Unnecessarily adding to bundle size